scheduler/src/cook/kubernetes/api.clj

```diff
@@ -800,7 +800,10 @@
         params-env (build-params-env parameters)
         progress-env (task/build-executor-environment job)
         checkpoint-env (checkpoint->env checkpoint)
-        main-env-base (merge environment params-env progress-env sandbox-env checkpoint-env)
+        metadata-env (cond-> {"COOK_COMPUTE_CLUSTER_NAME" compute-cluster-name
+                              "COOK_SCHEDULER_REST_URL" (config/scheduler-rest-url)}
+                       pool-name (assoc "COOK_POOL" pool-name))
+        main-env-base (merge environment params-env progress-env sandbox-env checkpoint-env metadata-env)
```

Comment:
Would it make sense to write an integration test for the URL and POOL environment variables?

Reply:
added